src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  decrease(damage) {
+    this.health -= damage;
```

Comment:
The 'decrease' method is a specific behavior related to the 'Carnivore' class. You should move this method to the 'Carnivore' class.